doc/en/api/.swagger-codegen-ignore

```diff
@@ -0,0 +1,23 @@
+# Swagger Codegen Ignore
```

Comment:
Do we actually need this file?

Reply:
We need a file (or the tool makes a needless  warming) because the folder is not created yet.